dev-docs/source/ReleaseChecklist.rst

```diff
@@ -309,55 +312,69 @@ Once the manual testing has passed (check with the Quality Assurance Manager):
 *  Disable release deploy jobs by building the
    `close-release-testing <https://builds.mantidproject.org/view/All/job/close-release-testing>`__
    job.
-*  On the ``release-next`` branch, update the git SHA for MSlice accordingly in
-   ``scripts/ExternalInterfaces/CMakeLists`` in case MSlice has to be updated.
-*  On the ``release-next`` branch, update major & minor versions accordingly in
-   ``buildconfig/CMake/VersionNumber.cmake``. Also uncomment ``VERSION_PATCH`` and
-   set it to ``0``.
+
+**Create the Release Candidates**
+
+We are now ready to create the release candidates ready for Smoke testing.
+
+*  On the ``release-next`` branch, create a PR to update the `git SHA
+   <https://github.com/mantidproject/mantid/blob/343037c685c0aca9151523d6a3e105504f8cf218/scripts/ExternalInterfaces/CMakeLists.txt#L11>`__
+   for MSlice.
+*  On the ``release-next`` branch, create a PR to update the `major & minor
+   <https://github.com/mantidproject/mantid/blob/master/buildconfig/CMake/VersionNumber.cmake>`__
+   versions accordingly. Also, uncomment ``VERSION_PATCH`` and set it to ``0``.
 *  Merge ``release-next`` branch back to ``master``
```

Comment:
This order is slightly incorrect, we should never have an uncommented `VERSION_PATCH` on the master branch and a PR is too slow. Instead it should be:

- Merge release-next branch back to master without pushing
- On master comment the `VERSION_PATCH` out again and commit
- Push the merged changes and extra commit to master
- Build the release kit....


I'll @martyngigg just to check this is correct, but AFAIK this is the only time we commit to master without a PR since we don't want to accidentally put out a nightly build labelled as a release from master whilst a PR goes through the motions.

Reply:
Thanks for improving this list. One issue with doing things directly on `master` is it reduces those who can perform the technical release manager role or bits of it anyway. I guess we have some of options:

1. Use PRs but make sure the technical release manager (TRM), if not a gatekeeper, pings a gatekeeper to merge the changes right away (if we were going to do them on master anyway then there is no difference)
2. Change the instructions to say ask a gatekeeper to make these alterations and push them
3. Restrict the TRM role to a gatekeeper (not keen on this as I don't think it's necessary and peope can't learn).

Both 1. & 2. require close communication with a gatekeeper so perhaps to lower the risk of the version number being left then we go with option 2?

Reply:
Ultimately, we're going to need a gatekeeper to either fast-track a PR through or merge straight to master before we close the release jobs on Jenkins (lest our nightlies go back a version). 
What about messaging a gatekeeper to be co-ordinate whether they'd prefer the fast PR or commit direct route? This keeps the TRM open to anyone but allows people to choose between 1/2 depending on how close we are to finishing up release